test/CssPrefixWindows.unittest.js

```
@@ -0,0 +1,183 @@
+"use strict";
+
+/** @typedef {{ version_added?: string | boolean | null, version_removed?: string | boolean | null, prefix?: string, alternative_name?: string, flags?: EXPECTED_ANY[] }} BcdSupport */
+/** @typedef {{ __compat?: { support: { [browser: string]: BcdSupport | BcdSupport[] } } }} BcdNode */
+
+const bcd =
+	/** @type {{ css: { properties: { [name: string]: BcdNode } } }} */ (
+		/** @type {unknown} */ (require("@mdn/browser-compat-data"))
+	);
+const { PREFIXED_PROPERTIES, PREFIX_BROWSERS } = require("../lib/css/data");
```

ESLint

[error] Missing file extension for "../lib/css/data"

(import/extensions)